### PR TITLE
Create Blog “streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible”

### PR DESCRIPTION
**Pull request by jullienl**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    margin: 20px;
+
+    box-shadow: 2px 2px 5px #ccc;
+
+  }
+
+</style>
+
+
+In the rapidly-evolving world of IT infrastructure management, achieving speed, efficiency, and reliability in server provisioning can make a significant difference. This is where cutting-edge tools like HPE Compute Ops Management and Ansible come into play. Together, they create a robust platform for managing your infrastructure seamlessly. In this blog post, we introduce an exciting new GitHub project that exemplifies how to harness these tools for optimal bare metal provisioning.
```
Comment (choukden) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:39]: Brand name for COM is "HPE GreenLake for Compute Ops Management"
see: https://www.hpe.com/us/en/hpe-greenlake-compute-ops-management.html 

So you may want to slightly update the 2nd sentence of this paragraph and other occurrences of COM throughout the post to be compliant with Brand naming.

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Lionel,

Thank you for your new blog post.
Just a minor suggestion for changes from me about naming convention for HPE GreenLake for COM.

Dale will go through the editorial review and you can expect additional suggestions for changes in the coming days or so.

Thanks,
Denis

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+### VMware ESXi  Bare Metal Provisioning on 2 x HPE ProLiant DL360 Gen10 Plus
+
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/_ySgROdd_Bw?si=CSzCklbTeRzaRtFg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Stay tuned as we will continue to update and maintain this project, incorporating user feedback and the latest advancements in technology.
+
+Get started now by visiting our [GitHub repository](https://github.com/jullienl/HPE-COM-baremetal), and begin transforming your server deployment strategy today!
+
+
```
Comment (Didier-Lalli) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:94]: It's a nice teaser, but how do I leverage this to use in MY environment. There could be a lot more to explain this like:

- Clone the repo
- then what?
-

**Review by Didier-Lalli — CHANGES_REQUESTED** [review]
Merci Lio. Je reste un peu sur ma faim. 
Ca met bien valeur le boulot que tu as réalisé mais pas trop comment je peux moi, client, l'adapter à mon environment. Peut-être tu pourrais rajouter un chapitre à ce propos.

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+- **Automated Provisions**: Kickstart your server setups without the tedious manual configurations.
+- **Centralized Control**: Manage your entire fleet of servers from a single pane of glass.
+- **Scalable Architecture**: Effortlessly scale your infrastructure to meet growing business demands.
+- **Pre-Built Playbooks**: Jumpstart your automation with curated collection, crafted for various deployment scenarios (Microsoft Windows Server, VMware ESXi and Red Hat Enterprise Linux).
+- **Custom Ansible Variables**: Enable you to define environment-specific parameters, ensuring that each server gets a configuration that fits its role in the infrastructure, ensuring that you have granular control over server provisioning.
+- **Comprehensive Documentation**: Detailed guides, videos and examples help you customize the workflow to your specific requirements.
+
+
+## Mastering Server Management with HPE GreenLake for Compute Ops Management
```
Comment (drensing) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:55]: Change to "Mastering server management with HPE GreenLake for Compute Ops Management."

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+HPE GreenLake for Compute Ops Management is a comprehensive solution for hardware resource management, providing a seamless way to handle server deployments. With its ability to manage health monitoring, orchestrate server configuration and firmware update workflows, and automate bare metal provisioning, administrators can ensure their data centers operate optimally with less effort and greater oversight. To learn more, see [HPE GreenLake for Compute Ops Management](https://www.hpe.com/emea_europe/en/hpe-greenlake-compute-ops-management.html)
+
+## Bridging HPE GreenLake for Compute Ops Management and Ansible
+
+HPE GreenLake for Compute Ops Management provides the foundational management capabilities essential for maintaining data center health and efficiency. When combined with the automation capabilities of Ansible, IT administrators can achieve unprecedented levels of automation.
+
+## Bringing it all together
+- **Automated Workflows**: Convert time-consuming manual processes into automated workflows that can be tracked and managed easily.
```
Comment (drensing) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:64]: Change to "Automated workflows:"

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+- **Scalable Infrastructure**: Embrace growth without compromising on performance or manageability.
```
Comment (drensing) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:65]: Change to "Scalable infrastructure: Embrace growth without compromising performance or manageability." (sentence case and remove the word "on")

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+- **Reduced Human Error**: Minimize mistakes by standardizing server configurations across the board.
```
Comment (drensing) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:66]: Change to "Reducing human error: "

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+## Mastering parallel execution with Ansible
+
+A key attribute of Ansible that I sought in this project is its impressive capability to execute tasks concurrently across numerous systems, thereby accelerating deployment processes. This feature is referred to as "forks" within Ansible. Set at a default of 5, the forks value is adjustable based on available system resources (CPU and memory), signifying that Ansible can carry out playbook tasks in parallel across five hosts from the inventory list. This parallel execution is among Ansible's standout functionalities, enhancing the effectiveness of bare-metal provisioning substantially. Moreover, this approach ensures consistent configurations across all provisioned hosts.
```
Comment (drensing) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:70]: Change the word "standout" to "outstanding" so it reads "...among Ansible's outstanding functionalities, enhancing..."

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+## Explore My Video Series
```
Comment (drensing) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:72]: Change to "Explore my video series"

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+
+### RHEL 9.3 Bare Metal Provisioning on 2 x HPE ProLiant DL360 Gen10 Plus
+
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/6_o8yB4cvag?si=OGQob5dNNF28rTF-" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+### VMware ESXi  Bare Metal Provisioning on 2 x HPE ProLiant DL360 Gen10 Plus
+
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/_ySgROdd_Bw?si=CSzCklbTeRzaRtFg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Join me on my [GitHub repository](https://github.com/jullienl/HPE-COM-baremetal), where a wealth of information awaits you in the README file. Learn how to effectively utilize this project, from cloning it into your environment to commencing its use, and witness the ways it can streamline your bare metal provisioning workflow.
```
Comment (drensing) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:88]: Change to "...to commencing **with** its use,..."

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+
+Stay tuned as I will continue to update and maintain this project, incorporating user [feedback](mailto:lio@hpe.com)   and the latest advancements that HPE GreenLake will offer.
+
+Get started now, and begin transforming your server deployment strategy today!
```
Comment (drensing) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:92]: Remove the comma

File: content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/6_o8yB4cvag?si=OGQob5dNNF28rTF-" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+### VMware ESXi  Bare Metal Provisioning on 2 x HPE ProLiant DL360 Gen10 Plus
+
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/_ySgROdd_Bw?si=CSzCklbTeRzaRtFg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Join me on my [GitHub repository](https://github.com/jullienl/HPE-COM-baremetal), where a wealth of information awaits you in the README file. Learn how to effectively utilize this project, from cloning it into your environment to commencing its use, and witness the ways it can streamline your bare metal provisioning workflow.
+
+Stay tuned as I will continue to update and maintain this project, incorporating user [feedback](mailto:lio@hpe.com)   and the latest advancements that HPE GreenLake will offer.
```
Comment (drensing) [inline content/blog/streamline-your-server-deployments-bare-metal-provisioning-with-hpe-compute-ops-management-and-ansible.md:90]: Take out the word "will" and just say "Stay tuned as I continue to update..."

**Review by drensing — COMMENTED** [review]
Great blog post, Jullien! Thank you!

Sorry it took me a bit longer to get to this.

All the best,
Dale

**Comment by jullienl** [comment]
I've just made a number of changes to the blog in response to Didier's pertinent comment

**Pull request merged**